src/panels/config/integrations/integration-panels/thread/thread-config-panel.ts

```diff
@@ -208,8 +208,15 @@ export class ThreadConfigPanel extends SubscribeMixin(LitElement) {
             ${network.routers.map((router) => {
               const otbr =
                 this._otbrInfo && this._otbrInfo[router.extended_address];
-              const showOverflow =
-                ("dataset" in network && router.border_agent_id) || otbr;
+              const showDefaultRouter = !!network.dataset;
+              const isDefaultRouter =
```

Comment:
Should also be used to display the icon in the row on line [245](https://github.com/home-assistant/frontend/pull/22183/files#diff-796b8571e876e5bf8f287e5037afd3219ca57fdb374673ff800e09d63104b087R245)